lib/jekyll-paginate-v2/generator/paginationPage.rb

```diff
@@ -41,6 +42,8 @@ def initialize(page_to_copy, cur_page_nr, total_pages, index_pageandext)
       end
 
       def set_url(url_value)
+        @path = url_value.delete_prefix '/'
```

Comment:
Caution: `String#delete_prefix` was added only in Ruby 2.5. Therefore, this will break on older Ruby versions.

Reply:
Does this seem better to you?

```ruby
if path[0] == '/'
  path[1..]
end
```